aspnetcore/blazor/components/css-isolation.md

```diff
@@ -0,0 +1,160 @@
+---
+title: ASP.NET Core Blazor CSS isolation
+author: daveabrock
+description: Learn how CSS isolation allows you to scope CSS to your components, which can simplify your CSS and avoid collisions with other components or libraries.
+monikerRange: '>= aspnetcore-5.0'
+ms.author: riande
+ms.custom: mvc
+ms.date: 09/17/2020
+no-loc: ["ASP.NET Core Identity", cookie, Cookie, Blazor, "Blazor Server", "Blazor WebAssembly", "Identity", "Let's Encrypt", Razor, SignalR]
+uid: blazor/components/css-isolation
+---
+# ASP.NET Core Blazor CSS isolation
+
+By [Dave Brock](https://twitter.com/daveabrock)
+
+Use cascading style sheets (CSS) isolation to only apply styles to the current component. CSS isolation simplifies an app's CSS footprint by preventing dependencies on global styles, and also helps to avoid styling conflicts between other components or libraries.
```

Comment:
Changed to:

CSS isolation simplifies an app's CSS footprint by preventing dependencies on global styles, and also helps to avoid styling conflicts between other components or libraries. Use CSS isolation to only apply styles to the current component.